# Upstream status: posted to the patch tracker

Subject: Fix iop321 cpuid
From: Dan Williams <dan.j.williams@intel.com>

commit a6a38a66224c7c578cfed2f584b440c81af0c3ae changed the iop321 id to a
value that does not work with all platforms. Change the mask to permit bit
11. Tested on an iq80321 600Mhz CRB.

http://www.arm.linux.org.uk/developer/patches/viewpatch.php?id=3780/1

Signed-off-by: Dan Williams <dan.j.williams@intel.com>
---

 arch/arm/mm/proc-xscale.S |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mm/proc-xscale.S b/arch/arm/mm/proc-xscale.S
index 561bff7..3ca0c92 100644
--- a/arch/arm/mm/proc-xscale.S
+++ b/arch/arm/mm/proc-xscale.S
@@ -644,7 +644,7 @@ __80219_proc_info:
 	.type	__8032x_proc_info,#object
 __8032x_proc_info:
 	.long	0x69052420
-	.long	0xffffffe0
+	.long	0xfffff7e0
 	.long   PMD_TYPE_SECT | \
 		PMD_SECT_BUFFERABLE | \
 		PMD_SECT_CACHEABLE | \

